src/basic/unit-file.c

```diff
@@ -567,8 +567,6 @@ int unit_file_build_name_map(
                                         if (!dst)
                                                 return log_oom();
                                 }
-
-                                log_debug("%s: normal unit file: %s", __func__, dst);
```

Comment:
Make it trace instead? For when things are desperate, more logs are better than fewer